.github/workflows/ci.yml

```diff
@@ -301,6 +302,14 @@ jobs:
           username: ${{ secrets.DOCKER_USR }}
           password: ${{ secrets.DOCKER_PSW }}
       
+      - name: Login to Upbound
+        uses: docker/login-action@v1
+        if: env.XPKG_ACCESS_ID != ''
+        with:
+          registry: xpkg.upbound.io
+          username: ${{ secrets.XPKG_ACCESS_ID }}
+          password: ${{ secrets.XPKG_TOKEN }}
```

Comment:
Do these need to be created at the crossplane-contrib org level? I don't see them.

Reply:
@negz I have created these tokens for `provider-aws` / `provider-gcp` / `provider-azure` in each individual repo with scope limited to the individual repositories -- happy to broaden permissions and set at org level if you prefer (I feel like we had this same conversation a few years ago 🙂 )